stdlib/contrib/anaisdg/print/print.flux

```diff
@@ -0,0 +1,23 @@
+// Package print provides function for converting values into tables
+// ## Metadata
+// introduced: NEXT
+package print
+
+
+import "array"
+
+// print function converts other data types to a table
+// ## Parameters
+// - val: The value you want to "print".
+// - result_name: The name you want your table to have.
+// ## Examples
+//
+// ### Perform a linear regression on a dataset
+// ```no_run
+// import "contrib/anaisdg/print"
+// import "print"
+//
+// < value = (sampledata.float() |> findRecord(fn: (key) => true,idx: 0))._value
+// >  print.print(val: value, result_name: "extracted value")
+// ```
+print = (val, result_name) => array.from(rows: [{"_value": display(v: val)}]) |> yield(name: display(v: result_name))
```

Comment:
I think `result_name` should have a default value of `_result` just so users don't have to specify a result name every time.

```suggestion
print = (val, result_name="_result") =>
    array.from(rows: [{"_value": display(v: val)}]) |> yield(name: display(v: result_name))
```

Reply:
You could also set `val` to receive piped-forward data. That way you could just pipe forward an extracted value into the function.

Reply:
Is there a way for it to receive both? I don't want to limit it to pipe forward in the case where people want to display variables.

Reply:
Absolutely. With this function definition:

```js
print = (val=<-, result_name="_result") =>
    array.from(rows: [{"_value": display(v: val)}]) |> yield(name: display(v: result_name))
```

Any of these will work:

```js
import "contrib/anaisdg/print"

123.4 |> print.print()
```

```js
import "contrib/anaisdg/print"

print.print(val: 123.4)
```

```js
import "contrib/anaisdg/print"
import "sampledata"

value = (sampledata.float() |> findRecord(fn: (key) => true,idx: 0))._value

value |> print.print()
```

```js
import "contrib/anaisdg/print"
import "sampledata"

value = (sampledata.float() |> findRecord(fn: (key) => true,idx: 0))._value

print.print(val: value)
```